.gitmodules

```diff
@@ -1,3 +1,10 @@
+[submodule "landing/public"]
+	path = landing/public
```

Comment:
I saw we use git submodule to refer to symphony-website repo and docs repo. Why we still need to keep the source code of doc site in symphony repo?